yarn-project/end-to-end/src/e2e_state_vars.test.ts

```diff
@@ -22,7 +22,7 @@ describe('e2e_state_vars', () => {
   beforeAll(async () => {
     ({ teardown, wallet, pxe } = await setup(2));
     contract = await DocsExampleContract.deploy(wallet).send().deployed();
-  }, 30_000);
+  }, 60_000);
```

Comment:
This started failing for me in CI but the code in this PR is not used in this test so it should not lead to slowdown.